io-ballerina/file_string_io.bal

```diff
@@ -35,6 +37,7 @@ public isolated function fileReadLines(@untainted string path) returns @tainted
 }
 
 # Reads file content as a stream of lines.
+# The resulting stream does not contain the terminal carriage(e.g. `\r` or `\n`).
```

Comment:
```suggestion
# The resulting stream does not contain the terminal carriage (e.g., `\r` or `\n`).
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-io/pull/199/commits/bb4a9a29bfe69ed3afc9a8921e5c4627e31efb9b